docs/components.md

```diff
@@ -57,6 +57,39 @@ Setting `exposedEvents` to `true` rather than an array will expose all events.
 
 Additional events can also be exposed by passing in an array of event names to `exposedEvents` on the component options object.
 
+Events on components are namespaced when exposed. This means the events are bound to only the model of the component regardless of what level in the applicaton it is being used.
```

Comment:
I think this can be clarified.  What does "bound to only the model of the component" mean?  Also, I'm not sure "only" is accurate here, since events can bubble without their namespace (exposed event `custom` on `my_component` can be heard with both `custom` and `custom:my_component`.